docs/source/markdown/podman-kube.1.md

```diff
@@ -14,7 +14,8 @@ file input.  Containers will be automatically started.
 
 | Command  | Man Page                                            | Description                                                                  |
 | -------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
-| play     | [podman-kube-play(1)](podman-kube-play.1.md)        | Create containers, pods or volumes based on Kubernetes YAML.                         |
+| down     | [podman-kube-down(1)](podman-kube-down.1.md)        | Remove containers or pods based on Kubernetes YAML.                          |
+| play     | [podman-kube-play(1)](podman-kube-play.1.md)        | Create containers, pods or volumes based on Kubernetes YAML.                 |
```

Comment:
Change to 
`Create pods, containers, and volumes based on Kubernetes YAML.

Reply:
It's probably worth updating the man pages in a separate PR.  This one inherits some warts from the already existing man pages.